SearchBar.js

```diff
@@ -1,15 +1,11 @@
-var NativeModules, PropTypes, RNSearchBar, React, SearchBar;
+import { Component, PropTypes } from 'react';
+import React, { NativeModules, requireNativeComponent } from 'react-native';
```

Comment:
As per the [0.25.1 release notes](https://github.com/facebook/react-native/releases/tag/v0.25.1), I think you should import `React` from the `'react'` module. The code above will still work, but is deprecated functionality.